src/misc/00_intro.rst

```diff
@@ -21,12 +21,16 @@ Glossary
   ICU
     **I**\ nternational **C**\ omponents for **U**\ nicode. ICU is an open-source project of mature C/C++ and Java libraries for Unicode support, software internationalization, and software globalization. `Learn More <http://site.icu-project.org/>`_. 
     
-    
+  R
+    **R** is a free software environment for statistical computing and graphics. It compiles and runs on a wide variety of UNIX platforms, Windows and MacOS. `Learn More <https://r-project.org/>`_.
+
+  CRAN
```

Comment:
This one, I really appreciate been added here, as I also had to look for this in the past.